AuroraEditor/Base/AppPreferences/UI/TerminalPreferences/TerminalPreferencesView.swift

```diff
@@ -79,6 +83,26 @@ public struct TerminalPreferencesView: View {
             }
         }
     }
+
+    private var cursorSelector: some View {
```

Comment:
Referenced the Terminal for this portion:
![image](https://github.com/AuroraEditor/AuroraEditor/assets/36312970/2678e69c-5fe3-4071-bd77-863d963cddef)